concepticondata/conceptlists/Oskolskaya-2021-254.tsv

```diff
@@ -0,0 +1,255 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Oskolskaya-2021-254-1	1	fire (n.)	221	FIRE
+Oskolskaya-2021-254-2	2	nose (n.)	1221	NOSE
+Oskolskaya-2021-254-3	3	go (v.)	695	GO
+Oskolskaya-2021-254-4	4	water (n.)	948	WATER
+Oskolskaya-2021-254-5	5	mouth (n.)	674	MOUTH
+Oskolskaya-2021-254-6	6	tongue (n.)	1205	TONGUE
+Oskolskaya-2021-254-7	7	blood (n.)	946	BLOOD
+Oskolskaya-2021-254-8	8	bone (n.)	1394	BONE
+Oskolskaya-2021-254-9	9	2SG pronoun	1215	THOU
+Oskolskaya-2021-254-10	10	root (n.)	670	ROOT
+Oskolskaya-2021-254-11	11	come (v.)	1446	COME
+Oskolskaya-2021-254-12	12	breast (n.) // (chest) (n.)	1402	BREAST
+Oskolskaya-2021-254-13	13	rain (n.)	658	RAIN (PRECIPITATION)
+Oskolskaya-2021-254-14	14	1SG pronoun	1209	I
+Oskolskaya-2021-254-15	15	name (n.)	1405	NAME
+Oskolskaya-2021-254-16	16	louse (n.)	1392	LOUSE
+Oskolskaya-2021-254-17	17	wing (n.)	1257	WING
+Oskolskaya-2021-254-18	18	meat (n.)	634	MEAT
+Oskolskaya-2021-254-19	19	arm / hand (n.)	2121	ARM OR HAND
+Oskolskaya-2021-254-20	20	fly (n.)	1504	FLY (INSECT)
+Oskolskaya-2021-254-21	21	night (n.)	1233	NIGHT
+Oskolskaya-2021-254-22	22	ear (n.)	1247	EAR
+Oskolskaya-2021-254-23	23	neck (n.)	1333	NECK
+Oskolskaya-2021-254-24	24	far	1406	FAR
+Oskolskaya-2021-254-25	25	do (make) (v.)	2575	DO OR MAKE
+Oskolskaya-2021-254-26	26	house (n.)	1252	HOUSE
+Oskolskaya-2021-254-27	27	stone (rock) (n.)	857	STONE
+Oskolskaya-2021-254-28	28	bitter	887	BITTER
+Oskolskaya-2021-254-29	29	say (v.)	1458	SAY
+Oskolskaya-2021-254-30	30	tooth (n.)	1380	TOOTH
+Oskolskaya-2021-254-31	31	hair (n.)	2648	HAIR (HEAD)
+Oskolskaya-2021-254-32	32	big	1202	BIG
+Oskolskaya-2021-254-33	33	one	1493	ONE
+Oskolskaya-2021-254-34	34	who?	1235	WHO
+Oskolskaya-2021-254-35	35	3SG pronoun	262	HE OR SHE OR IT
+Oskolskaya-2021-254-36	36	hit / beat (v.)	1665	BEAT
```

Comment:
Maybe rather `2133 STRIKE OR BEAT`?

Reply:
Please make sure to compare that it is the same as in Savelyev, since the concepts are supposed to be the same here.

Reply:
It's mapped to BEAT in Savelyev-2020-254.

Reply:
I used Savelyev-2020-254 as the mapping source, so all the entries will match S2020.